tests/utils/collections/userAccount.ts

```diff
@@ -0,0 +1,16 @@
+import CONST from '@src/CONST';
+import type {Account} from '@src/types/onyx';
+
+function getValidAccount(credentialLogin = ''): Account {
+    return {
+        validated: true,
+        primaryLogin: credentialLogin,
+        isSAMLRequired: false,
+        isSAMLEnabled: false,
+        isLoading: false,
+        requiresTwoFactorAuth: false,
+    } as Account;
```

Comment:
Is the cast really necessary?

Reply:
It isn't, just helped me at first with auto-complete

Reply:
Then remove it, please! Don't ever leave unnecessary casts.